src/main/java/nl/jiankai/mapper/converters/ArrayToStringArrayConverter.java

```
@@ -0,0 +1,32 @@
+package nl.jiankai.mapper.converters;
+
+import nl.jiankai.annotations.Converter;
+
+import java.sql.Array;
+import java.sql.SQLException;
+
+@Converter(autoApply = true)
+public class ArrayToStringArrayConverter implements AttributeConverter<Array, String[]> {
+
+    @Override
+    public String[] convert(Array value) {
+        String[] convertedValue = new String[0];
+    try {
+        convertedValue = (String[]) value.getArray();
+    }
+    catch (SQLException e){
+        System.out.println(e.getCause().getMessage());
```

Use a logger please. Avoid souts. Also we shouldn't silently fail when the converter can not convert the value. The mapper should know and handle this.